main.tf

```diff
@@ -33,14 +33,18 @@ resource "aws_lambda_function" "lambda_application" {
   role          = aws_iam_role.lambda_application_execution_role.arn
   handler       = each.value.handler
 
+
+  publish     = var.publish
```

Comment:
thought we were removing this out of a variable? i.e `publish = true`

Reply:
Publish true is done in terraform modules. There is an option in modules if someone wants to have an alias without specific version